src/client/components/insurance-information/AdditionalInformationSection.jsx

```diff
@@ -81,7 +81,7 @@ class AdditionalInformationSection extends React.Component {
 function mapStateToProps(state) {
   return {
     data: state.veteran,
-    isSectionComplete: state.uiState.sections['/personal-information/additional-information'].complete
+    isSectionComplete: state.uiState.sections['/insurance-information/va-facility'].complete
```

Comment:
Since this section is now called `va-facility` in the path, should we rename the component?